app/views/items/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<section class="page-wrapper">
+  <div class="page-title-bar">
+    <span class="page-title">Add a New Item for <%= @merchant.name %></span>
+  </div>
+  <%= form_tag "/merchants/#{@merchant.id}/items", method: :post, class: "margin" do %>
+    <p>Name:        <br/><%= text_field_tag :name %></p>
+    <p>Description: <br/><%= text_area_tag :description %></p>
+    <p>Price:       <br/><%= text_field_tag :price %></p>
+    <p>Image:       <br/><%= text_field_tag :image %></p>
+    <p>Inventory:   <br/><%= text_field_tag :inventory %></p>
+    <p><%= submit_tag 'Add Item'%></p>
```

Comment:
For any forms you do, if you want labels, you can also look into the label_tag. Looks like you may be doing this for format, but you can always format any label tags you have.